okhttp/src/jvmTest/kotlin/okhttp3/DuplexTest.kt

```diff
@@ -69,10 +72,17 @@ class DuplexTest {
 
   private var listener = RecordingEventListener()
   private val handshakeCertificates = platform.localhostHandshakeCertificates()
+  private val loggingInterceptor =
+    HttpLoggingInterceptor({
+      // Not testing output here, but making sure it doesn't fail
+    }).apply {
+      level = Level.NONE
+    }
```

Comment:
Because this interceptor now changes where decompression happens, the rest of this test is like, adversely impacted?